string name
string value
---
uint8 result
